python/meterpreter/ext_server_stdapi.py

```diff
@@ -2548,3 +2553,15 @@ def stdapi_ui_get_idle_time(request, response):
     idle_time = (GetTickCount() - info.dwTime) / 1000
     response += tlv_pack(TLV_TYPE_IDLE_TIME, idle_time)
     return ERROR_SUCCESS, response
+
+@register_function_if(has_termios and has_fcntl)
+def stdapi_set_term_size(request, response):
+    channel_id = packet_get_tlv(request, TLV_TYPE_CHANNEL_ID)['value']
+    tlvs = list(packet_enum_tlvs(request, TLV_TYPE_UINT))
+    rows = tlvs[0]['value']
+    columns = tlvs[1]['value']
+    if channel_id in meterpreter.interact_channels:
+        proc_h = meterpreter.channels[channel_id].proc_h
+        winsize = struct.pack("HHHH", rows, columns, 0, 0)
```

Comment:
hmm sorry but I don't understand, I thought that since python meterpreter was portable you'd always want it to use the native endianness? I'm probably not understanding something correctly though

Reply:
You're right, I had misread this as sending the packed value back to Metasploit in which case they would both need to be the same endianness but since you're using it for `fcntl.ioctl` it's correct and should be the native.